.github/workflows/ci.yml

```diff
@@ -173,6 +173,7 @@ jobs:
       skip-pre-commits: ${{needs.build-info.outputs.skip-pre-commits}}
       canary-run: ${{needs.build-info.outputs.canary-run}}
       latest-versions-only: ${{needs.build-info.outputs.latest-versions-only}}
+      enable-aip-44: "true"
```

Comment:
Which means we are going to run tests in DB isolation mode only? Should we run them in both mode?

Reply:
Ah. Yeah. I think we can remove it now - those are basic tests (docs/api generation etc. - they do not run the "unit tests" - we